xml_inscriptions/transcribed/NY.NY.MMA.G.74.51.5884.xml

```diff
@@ -0,0 +1,183 @@
+                            <placeName ref="europe.greece.cyprus.kition">Larnaca (Kition), Cyprus</placeName>
+                        </origin>
+                        <provenance type="transferred" subtype="acquired">
+                            <p>
+                                <orgName>The <persName role="owner"><name key="Cesnola">Cesnola</name></persName> Collection</orgName>, Purchased by subscription, <date notBefore="1873" notAfter="1876">1874–76</date>
+                                
+                            </p>
+                        </provenance>
+                        
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Cesnola"/>
+                        <biblScope unit="item">no. 61</biblScope>
+                    </bibl>
+                    <bibl> 
+                        <ref type="eternal" target="https://www.metmuseum.org/art/collection/search/244756">Metropolitan Museum Web Entry</ref> 
+                    </bibl>
+                    
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+        
+        <encodingDesc>
+            
+            <xi:include
+                href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for US Epigraphy controlled values</p>
+                </xi:fallback>
+            </xi:include>
+            
+        </encodingDesc>
+        
+        <profileDesc> <!-- not used --> </profileDesc>
+        
+        
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+        <!-- Record all changes here -->
+        
+        <revisionDesc>
+            <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+            <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+            <change when="2020-12-12" who="Itamar Levin">Transcription, translation, and
+                metadata</change>
+        </revisionDesc>
+        
+    </teiHeader>
+    
+    <!--  *************************************  <facsimile>  *********************************  -->
+    <!-- Used to record images of an inscription.   -->
+    
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url=""/>
+        </surface>
+    </facsimile>
+    
+    <text>
+        <body>
+            <div type="edition">
+                <ab>
+                    <lb n="1"/><persName><name key="Ζωὶλονης">Ζωὶλονη</name></persName> χρη
```

Comment:
Is the key right here? Based on what you have in the English, it should be Ζωίλονη, no? Also, watch your accentuation in the name.